views/accessibility-statement.html

```diff
@@ -18,120 +18,87 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
 
-        <h1 class="govuk-heading-l">
-            Accessibility statement for the Remove a company director service
+        <h1 class="govuk-heading-xl">
```

Comment:
Is this intended?

Reply:
Yes, it matches the new prototype version